### ISSUE TITLE
App keeps getting numbers in negative

### ISSUE THREAD

**Issue by hisham2007** [AndlyticsProject/andlytics#713]
App keeps getting numbers in negative (minus)

**Comment by willlunniss** [comment]
Need more details such as the numbers of the dev console vs the app.

**Comment by hisham2007** [comment]
my app is com.tabe3.news
i face this issue also with all my apps
when there is a new comment
change in comments count is -1

**Comment by willlunniss** [comment]
Do the total numbers match? There was a recent bug for the comments that has been fixed in master but not published yet.

**Comment by hisham2007** [comment]
give me your mail to send you screenshot
or i can wait unill you publish new update

**Comment by TheNephilim88** [comment]
Any update here, especially as the comment-bug has been fixed?

**Comment by hisham2007** [comment]
yup. it was fixed. but it returned back 2 days again and even bigger. my app "Aio Remote" is downloaded 800 000 times. but now in Andlytics, it displays -424232 !!!  and display actual downloads as 393407

also all installs changes are negative

**Comment by willlunniss** [comment]
Please check the total numbers on the Google Developer Console. A number of my apps lost ~1/3 of their downloads a few days ago and then they came back again. Andlytics only does a diff against the last day so if the totals on the dev console drop then you will get negative numbers.

**Comment by johnjohndoe** [comment]
Same as #712.

---

Negative download stats occur when the **system date** of the device is set **to the future** and the app triggers a download afterwards. This is a normal scenario for a developer who uses the device to debug application behavior with various dates. I had this happening to myself a lot.
Therefore, I recommend do not use the local date and time settings. When I manually set it to some future date statistics are saved with that timestamp. Use server dates instead.
